src/openrct2/core/Guard.cpp

```diff
@@ -78,15 +78,10 @@ namespace Guard
         Console::Error::WriteLine("Version: %s", gVersionInfoFull);
 
         // This is never freed, but acceptable considering we are about to crash out
-        utf8* formattedMessage = nullptr;
+        std::string_view formattedMessage = String::Format_VA(message, args);
```

Comment:
This is not what I asked for, nor does it fix the issue. The builds now even fail. And what if message is null? No guarantees that the function handles it properly.

Reply:
Addressed issue by instead changing formattedMessage to std::string and static casting formattedMessage to std::string_view